Chap_API_Struct.tex

```diff
@@ -121,510 +125,116 @@ \subsubsection{General Error Constants}
 \declareconstitem{PMIX_ERROR}
 General Error
 %
-\declareconstitem{PMIX_ERR_SILENT}
```

Comment:
Maybe I missed them, but a lot of  these error codes removed here do not appear to be defined somewhere else.

Reply:
They aren't because they are not returned to the user